src/main/thrift/parquet.thrift

```diff
@@ -242,43 +242,36 @@ struct SizeStatistics {
  * All fields are optional.
  */
 struct Statistics {
-   /**
-    * DEPRECATED: min and max value of the column. Use min_value and max_value.
-    *
-    * Values are encoded using PLAIN encoding, except that variable-length byte
-    * arrays do not include a length prefix.
-    *
-    * These fields encode min and max values determined by signed comparison
-    * only. New files should use the correct order for a column's logical type
-    * and store the values in the min_value and max_value fields.
-    *
-    * To support older readers, these may be set when the column order is
-    * signed.
-    */
+   /* DEPRECATED: do not use */
    1: optional binary max;
    2: optional binary min;
    /** count of null value in the column */
    3: optional i64 null_count;
    /** count of distinct values occurring */
    4: optional i64 distinct_count;
    /**
-    * Lower and upper bound values for the column, determined by its ColumnOrder.
+    * Only one pair of max_value/min_value, max1/min1, max2/min2, max4/min4,
```

Comment:
this replicates T1 can it be left off here?

Reply:
Removed T1 from this PR.